addons/portal_rating/static/src/js/portal_rating_composer.js

```diff
@@ -56,17 +56,54 @@ var RatingPopupComposer = publicWidget.Widget.extend({
 
 publicWidget.registry.RatingPopupComposer = publicWidget.Widget.extend({
     selector: '.o_rating_popup_composer',
+    custom_events: {
+        reload_rating_popup_composer: '_onReloadRatingPopupComposer',
+    },
 
     /**
      * @override
      */
     start: function () {
-        var ratingPopupData = this.$el.data();
-        var ratingPopup = new RatingPopupComposer(this, ratingPopupData);
+        this.ratingPopupData = this.$el.data();
+        this.ratingPopupData.display_composer = !(this.ratingPopupData.disable_composer && !session.is_website_user);
```

Comment:
> as mentioned before: `!this.ratingPopupData.disable_composer && session.is_website_user`

this expression evaluates to false whether any user logged in or not(public user).
so, I changed to this.ratingPopupData.disable_composer && !session.is_website_user
if public user(any user not logged in) it will evaluate to false,
if user is logged in it will evaluate to true

Reply:
> as mentioned before: `!this.ratingPopupData.disable_composer && session.is_website_user`

Hello, this expression always evaluates to false (wether any user logged in or not)
hence I changed expression to  
!(this.ratingPopupData.disable_composer && session.is_website_user)
this will be true if any user is logged in, and false otherwise.

Reply:
Seems I was mistaken about the meaning of session.is_website_user, sorry about that. Still don't need parentheses though, `!(this.ratingPopupData.disable_composer || session.is_website_user)` is logically equivalent to `!this.ratingPopupData.disable_composer && !session.is_website_user`